ciscosma.json

```
@@ -0,0 +1,2297 @@
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.senderAddresses",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.recipientList",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary.list_type",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary.view_by",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary.status",
+                    "data_type": "string",
+                    "column_name": "Status",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "delete list entry",
+            "identifier": "delete_list_entry",
+            "description": "Delete an entry from the spam quarantine safelist or blocklist",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "list_type": {
+                    "description": "Type of list to delete entry from",
+                    "data_type": "string",
+                    "required": true,
+                    "order": 0,
+                    "value_list": ["safelist", "blocklist"],
+                    "default": "safelist"
+                },
+                "view_by": {
+                    "description": "Delete entry by sender or recipient",
+                    "data_type": "string",
+                    "order": 1,
+                    "value_list": ["sender", "recipient"],
+                    "default": "recipient"
+                },
+                "recipient_list": {
+                    "description": "Recipient email addresses (comma-separated if multiple) - Required with view_by=recipient",
+                    "data_type": "string",
+                    "order": 2
+                },
+                "sender_list": {
+                    "description": "Sender addresses or domains (comma-separated if multiple) - Required with view_by=sender",
+                    "data_type": "string",
+                    "order": 3
+                }
```

Why not `email_addresses`?
```suggestion
                "email_addresses": {
                    "description": "Sender or recipient email addresses, dependent upon the 'view_by' parameter. Comma-separated if multiple",
                    "required": true,
                    "contains": ["email"],
                    "data_type": "string",
                    "order": 2
                }
```

---

Same reason as listed above, we can change if that's what we think is best.